amoro-mixed-format/amoro-mixed-format-flink/amoro-mixed-format-flink-common/src/main/java/org/apache/amoro/flink/interceptor/ProxyFactory.java

```diff
@@ -23,7 +23,7 @@
 import java.io.Serializable;
 
 /**
- * Create proxy in runtime to avoid 'ClassNotFoundException: XXX$$EnhancerByCglib'
```

Comment:
do we change this because of the format checker?

Reply:
Yes.